src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -12,8 +12,10 @@
 
 
     public static final String MESSAGE_PHONE_CONSTRAINTS =
-            "Phone numbers can only contain numbers, and should be at least 3 digits long";
-    public static final String PHONE_VALIDATION_REGEX = "\\d{3,}";
+            "Users are to label their numbers with HP: (HandPhone number) H: (Home number) or O: (Office number)\n "
+                    + "For example, HP: 91234567 H: 97287288 O: 34987987.\n "
+                    + "Phone numbers can only contain numbers, and should be at least 3 digits long";
+    public static final String PHONE_VALIDATION_REGEX = "(?=(?:.*\\d){3})[(HPO:)0-9 ]+";
```

Comment:
I'm not very familiar with REGEX, but I was just wondering what happens if the user has multiple white spaces in between the label and the number?
e.g. "HP: ___________ 45679213" 
( _ represents whitespace)

Reply:
Will update my regex to take into account consecutive whitespaces